### PR TITLE
Ux revamp/analysis panel layer download

### PR DESCRIPTION
**Pull request by ajanim**
UX Revamp- Analysis Panel + Layer Download

How to test the feature:

- [ ] AAUser, on the analysis view, I am able to select parameters for an analysis.
- [ ] AAUser, on the analysis view, once I have the analysis results, I can also view it as a table by clicking the "View Table" button.
- [ ] AAUser, on the analysis view, once I have the analysis results, I can click "Clear Analysis," which then clears map view/table view and reset the the analysis parameters I had previously selected.
- [ ] AAUser, on the layers tab, once I have selected a layer, I can download it as a CSV/GeoJSON/GeoTiff.

Screenshot/video of feature:

### CODE REVIEW

File: frontend/src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -116,6 +116,7 @@ const styles = (theme: Theme) =>
       height: '7vh',
       display: 'flex',
       justifyContent: 'center',
+      // height: '7%',
```
Comment (sifelix) [inline frontend/src/components/NavBar/index.tsx:119]: Remove the comment

File: frontend/src/components/MapView/LeftPanel/index.tsx [hunk 2]
```diff
@@ -3,23 +3,32 @@ import {
...
+interface StyleProps {
+  isPanelExtended: boolean;
+}
+
+const useStyles = makeStyles<Theme, StyleProps>(() =>
   createStyles({
     paper: {
       marginTop: '7vh',
-      width: '30%',
+      height: '93%',
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/index.tsx:22]: would it be safer to use vh unit?

File: frontend/src/components/MapView/LeftPanel/LeftPanelTabs.tsx [hunk 3]
```diff
@@ -65,18 +66,27 @@ interface TabsProps {
...
 
   const handleChange = (_: any, newValue: number) => {
     setValue(newValue);
   };
 
   return (
-    <div className={classes.root}>
+    <div
+      className={classes.root}
+      style={{ width: isPanelExtended ? '50%' : 'auto' }}
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/LeftPanelTabs.tsx:88]: why don't you conditionally set it in the makeStyle?

File: frontend/src/components/MapView/LeftPanel/LeftPanelTabs.tsx [hunk 4]
```diff
@@ -38,8 +39,8 @@ function a11yProps(index: any) {
 const useStyles = makeStyles(() =>
   createStyles({
     root: {
-      width: '100%',
-      height: '100%',
+      width: 'auto',
+      height: 'auto',
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/LeftPanelTabs.tsx:43]: Why do you need auto here?

File: frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx [hunk 5]
```diff
@@ -0,0 +1,819 @@
...
+  safeDispatchAddLayer,
+} from '../../../../utils/map-utils';
+import { removeLayer } from '../../../../context/mapStateSlice';
+import { DEFAULT_DATE_FORMAT } from '../../../../utils/name-utils';
+import LayerDropdown from '../../Layers/LayerDropdown';
+import SimpleDropdown from '../../../Common/SimpleDropdown';
+import AnalysisTable from '../../Analyser/AnalysisTable';
+import { Extent } from '../../Layers/raster-utils';
+
+function AnalysisPanel({
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx:86]: Can you delete the equivalent code in Analyser/index.tsx that is not used anymore?

File: frontend/src/components/MapView/LeftPanel/layersPanel/MenuSwitch/SwitchItem/index.tsx [hunk 6]
```diff
@@ -199,6 +292,42 @@ function SwitchItem({ classes, layer }: SwitchItemProps) {
...
+              </MenuItem>
+            </>
+          )}
+          {layer.type === 'wms' &&
+            layer.baseUrl.includes('api.earthobservation.vam.wfp.org/ows') && (
+              <MenuItem onClick={handleDownloadGeoTiff}>
+                Download as GeoTIFF
+              </MenuItem>
+            )}
+        </Menu>
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/layersPanel/MenuSwitch/SwitchItem/index.tsx:330]: Can you extract this in a new component?

File: frontend/src/components/MapView/LeftPanel/layersPanel/MenuSwitch/SwitchItem/index.tsx [hunk 7]
```diff
@@ -199,6 +292,42 @@ function SwitchItem({ classes, layer }: SwitchItemProps) {
...
+        <Menu
+          id="download-menu"
+          anchorEl={downloadMenuAnchorEl}
+          keepMounted
+          open={Boolean(downloadMenuAnchorEl)}
+          onClose={handleDownloadMenuClose}
+        >
+          <MenuItem onClick={handleDownloadMenuClose}>
+            Exposure Analysis
+          </MenuItem>
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/layersPanel/MenuSwitch/SwitchItem/index.tsx:315]: not working?

File: frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx [hunk 8]
```diff
@@ -0,0 +1,819 @@
...
+                onClick={() => setIsPanelExtended(!isPanelExtended)}
+                className={classes.analysisTableCloseButton}
+              >
+                <CloseRounded />
+              </IconButton>
+            </div>
+            <AnalysisTable
+              tableData={analysisResult.tableData}
+              columns={translatedColumns}
+            />
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx:697]: AnalysisTable and th remaining content of the analysis panel should be two elements of a flex-direction row. Only the table section should be scrollable

File: frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx [hunk 9]
```diff
@@ -0,0 +1,819 @@
...
+import { DEFAULT_DATE_FORMAT } from '../../../../utils/name-utils';
+import LayerDropdown from '../../Layers/LayerDropdown';
+import SimpleDropdown from '../../../Common/SimpleDropdown';
+import AnalysisTable from '../../Analyser/AnalysisTable';
+import { Extent } from '../../Layers/raster-utils';
+
+function AnalysisPanel({
+  extent,
+  isPanelExtended,
+  setIsPanelExtended,
```
Comment (sifelix) [inline frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx:89]: If AnalysisPanel is unmounted the panel should not be extended anymore

**Comment by wadhwamatic** [comment]
@ajanim - great work on this! Some initial feedback:

1. The text for the headers in left panel (layers, charts, analysis) as well as the layer actions 'exposure analysis' and 'download as geotiff' aren't translating. I added translations for Mozambique in this branch to test and verify this
2. The analysis result table should display on top of the timeline
3. After I clear an analysis, the date value disappears and it's not intuitive that the calendar icon is a link to repopulate that. Can we have it default back to the latest available date? This would be consistent with current behavior. We may want to consider making it clearer that the date value is an input field (I see the Figma design missed this, sorry)
4. I'm hoping we can prioritize collapsing the sidebar as it takes a lot of screen space away from the map. See the top row of the Figma design for this

See screenshots for issues 1-3 below

> 1. missing translations
Screen Shot 2023-01-03 at 18 06 51

> 2. table view should appear in front of timeline
Screen Shot 2023-01-03 at 18 44 35

> 3. date value input isn't clear after clearing an analysis
Screen Shot 2023-01-03 at 18 45 16

File: frontend/src/components/MapView/Layers/LayerDropdown.tsx [hunk 10]
```diff
@@ -16,16 +18,41 @@ import {
...
+  createStyles({
+    selectRoot: {
+      color: 'black',
+      minWidth: '400px',
+      maxWidth: '450px',
+      width: 'auto',
+      '& label': {
+        color: '#333333',
+      },
+      '& .MuiInputBase-root': {
```
Comment (ericboucher) [inline frontend/src/components/MapView/Layers/LayerDropdown.tsx:31]: Should this be handled by passing another class to InputProps?
Not a blocker but seeing many specific `.Mui`  cases is often an orange flag

**Review by ericboucher — APPROVED** [review]
Just a minor comment on MUI styling. Code looks great otherwise, nice work.

File: frontend/src/components/MapView/DateSelector/index.tsx [hunk 11]
```diff
@@ -340,8 +340,7 @@ const styles = (theme: Theme) =>
     container: {
       position: 'absolute',
       bottom: '8%',
-      zIndex: theme.zIndex.modal,
-      width: '100vw',
+      width: '100%',
```
Comment (ericboucher) [inline frontend/src/components/MapView/DateSelector/index.tsx:343]: Can you explain this change? There might be a reason why we were using 100vw initially

**Pull request merged**